dist/mushroom-strategy.js

```diff
@@ -61,65 +85,99 @@ const getFilteredEntitiesFromStates = (info, entities, devices, area, startsWith
-        {
-          type: "horizontal-stack",
-          cards: [
-            {
-              type: "custom:mushroom-template-card",
-              icon: iconOff,
-              layout: "vertical",
-              icon_color: "red",
-              tap_action: {
-                action: "call-service",
-                service: offService,
-                target: {
-                  area_id: area_id,
-                },
-                data: {},
+/**
+ * Create a title card with controls to switch the entities of given areas.
+ *
+ * A title card is...
+ * TODO: Correct/Complete function description.
+ *
+ * @param {string} title Title of the card.
+ * @param {string} subtitle Subtitle of the card.
+ * @param {string} offService Name of service to switch off the group.
+ * @param {string} onService Name of service to switch on the group.
+ * @param {string} iconOff Icon to set when group is switched off.
+ * @param {string} iconOn Icon to set when group is switched on.
+ * @param {string|string[]} area_id Id of the areas(s) to switch.
+ *
+ * @return {Object} A title card object.
+ */
+function createTitleCard(title, subtitle, offService, onService, iconOff, iconOn, area_id) {
+  return {
+    type: "horizontal-stack",
+    cards: [
+      {
+        type: "custom:mushroom-title-card",
+        title: title,
+        subtitle: subtitle,
+      },
+      {
+        type: "horizontal-stack",
+        cards: [
+          {
+            type: "custom:mushroom-template-card",
+            icon: iconOff,
+            layout: "vertical",
+            icon_color: "red",
+            tap_action: {
+              action: "call-service",
+              service: offService,
+              target: {
+                area_id: area_id,
               },
+              data: {},
             },
-            {
-              type: "custom:mushroom-template-card",
-              icon: iconOn,
-              layout: "vertical",
-              icon_color: "amber",
-              tap_action: {
-                action: "call-service",
-                service: onService,
-                target: {
-                  area_id: area_id,
-                },
-                data: {},
+          },
+          {
+            type: "custom:mushroom-template-card",
+            icon: iconOn,
+            layout: "vertical",
+            icon_color: "amber",
+            tap_action: {
+              action: "call-service",
+              service: onService,
+              target: {
+                area_id: area_id,
               },
+              data: {},
             },
-          ],
-        },
-      ],
-    }
-);
+          },
+        ],
+      },
+    ],
+  };
+}
 
-const createPlatformCard = (entities, entity_config, defaultCard, titleCard, doubleTapActionConfig) => {
+/**
+ * Create a platform card for each given entity.
+ *
+ * A platform card is a... and consists of an optional title-card and an entity-card.
+ * TODO: Correct/Complete function description.

```

Comment:
@AalianKhan Please follow up line 154.

Reply:
That function creates a list of cards from the given entities.
Should it be renamed to something more accurate such as `createListOfCards`?

Reply:
Don't worry about it too much. Maybe we'll want to change the name to something more clear while making progress with optimizing the code.